tests/unit/src/test/scala/tests/DocumentSymbolSlowSuite.scala

```diff
@@ -0,0 +1,92 @@
+package tests
+
+object DocumentSymbolSlowSuite extends BaseSlowSuite("documentSymbol") {
+
+  testAsync("documentSymbol") {
+    for {
+      // start with code that does not parse (notice the first char in Main.scala)
+      _ <- server.initialize(
+        """|
+           |/metals.json
+           |{
+           |  "a": { }
+           |}
+           |/a/src/main/scala/a/Main.scala
+           |}package a
```

Comment:
intended curly brace?

Reply:
see it's intended now, would be good to have something clearer

```scala
} // <- parse error
```

Reply:
Makes sense, I'll add it